source/docs/api-reference/wpilib-integration/motorcontroller-integration.rst

```diff
@@ -0,0 +1,79 @@
+MotorController Integration
+===========================
+
+Phoenix Pro motor classes such as ``TalonFX`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html>`__) implement the ``MotorController`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_motor_controller.html>`__) interface.
+
+Phoenix Pro implements all inherited functions:
+
+- ``set``
+- ``setVoltage``
+- ``get``
+- ``setInverted``
+- ``getInverted``
+- ``disable``
+- ``stopMotor``
```

Comment:
I don't think we should list off every function in the MotorController API, as we shouldn't have to update our docs when functions are added to or removed from the WPILib API. We already link to the API docs, and since it's an interface, we *have* to have implementations for all of those functions.